partner_title_order/tests/test_partner_title.py

```diff
@@ -1,10 +1,10 @@
 # Copyright 2020 Camptocamp SA
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
 
-from odoo.tests.common import SavepointCase
+import odoo.tests.common as common
 
 
-class TestPartnerTitle(SavepointCase):
+class TestPartnerTitle(common.TransactionCase):
```

Comment:
@Kimkhoi3010 FYI, to avoid the tracking_disable context change here under, you can use BaseCommon class from base addon instead.

Reply:
hello @rousseldenis thanks for your time.

Reply:
Hello @Kimkhoi3010 , I think @rousseldenis was talking about this base class: https://github.com/odoo/odoo/blob/18.0/odoo/addons/base/tests/common.py#L21 (that is leveraging the `DISABLED_MAIL_CONTEXT` keys automatically)

I cannot see this change with your push, can you check please?